Dockerfile

```diff
@@ -8,4 +8,4 @@ COPY watches.yaml ${HOME}/watches.yaml
 COPY roles/ ${HOME}/roles/
 COPY playbooks/ ${HOME}/playbooks/
 
-ENTRYPOINT ["/tini", "--", "/usr/local/bin/ansible-operator", "run", "--watches-file=./watches.yaml", "--reconcile-period=0s"]
+ENTRYPOINT ["/tini", "--", "/usr/local/bin/ansible-operator", "run", "--watches-file=./watches.yaml", "--reconcile-period=0s", "--max-concurrent-reconciles", "2"]
```

Comment:
so how does this interact with the environment variables?

Reply:
@kdelee I tested it out and if an environment variable like the following is set, it will override the Dockerfile change.

```
    "name": "MAX_CONCURRENT_RECONCILES_ANSIBLEJOB_TOWER_ANSIBLE_COM",
    "value": "10"
```

Furthermore, it changes the number of workers for that particular resource.